src/main/java/survey/Question.java

```diff
@@ -1,6 +1,15 @@
 package survey;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
 public abstract class Question {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
Don't we require an accessory for the ID field (public Integer getID())

Reply:
There is a n accessor.